app/views/jobseekers/job_applications/build/catholic_following_religion.html.slim

```diff
@@ -20,8 +20,8 @@
       = f.govuk_error_summary
 
       = f.govuk_radio_buttons_fieldset :following_religion do
-        = f.govuk_radio_button :following_religion, true, link_errors: true, label: { text: t("helpers.label.jobseekers_job_application_following_religion_form.following_religion_options.yes") }
-        = f.govuk_radio_button :following_religion, false, label: { text: t("helpers.label.jobseekers_job_application_following_religion_form.following_religion_options.no") }
+        = f.govuk_radio_button :following_religion, true, link_errors: true
```

Comment:
This is `boolean` and the `"false"` bellow is a `string`. The `"true"` in the non-catholic view is also a string.

Is this ok?